synapse/datamodel.py

```diff
@@ -167,16 +197,49 @@ def addTufoForm(self, form, **info):
 
     def isTufoForm(self, name):
         '''
-        Returns True if the given name is a form.
+        Check if a form is a valid form.
+
+        Args:
+            name (str): Form to check
+
+        Returns:
+            bool: True if the form is a valid form. False otherwise.
         '''
         return name in self.forms
 
+    def reqTufoForm(self, name):
```

Comment:
Nice catch in https://github.com/vertexproject/synapse/pull/506

Reply:
i first devved it in this branch then rebased the commit out so it could be cherry picked over into #506 since i needed it there.